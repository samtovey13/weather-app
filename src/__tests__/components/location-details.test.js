import React from 'react';
import { render } from "@testing-library/react";
import LocationDetails from '../../components/location-details';

const city = "Manchester";
const country = "UK";

it('renders correctly', () => {
  const { asFragment } = render(<LocationDetails city={city} country={country} />);
  expect(asFragment()).toMatchSnapshot();
})

it("renders the correct city and location props", () => {
  const { getByText } = render(
    <LocationDetails city={city} country={country} />
  );

  expect(getByText("Manchester, UK")).toHaveClass("location-details");
});