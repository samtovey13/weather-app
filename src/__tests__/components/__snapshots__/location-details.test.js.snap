// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<DocumentFragment>
  <h1
    class="location-details"
  >
    Manchester, UK
  </h1>
</DocumentFragment>
`;
